### DISCUSSION TITLE
Split Movies / TV Shows in Netflix My List

### DISCUSSION THREAD

**Discussion by tsioukas** [CastagnaIT/plugin.video.netflix#1560] [Ideas]
Is it possible to add a feature to split Movies and TV Shows in Netflix MyList? I know that Netflix doesn’t do it, but it will be a good way for someone to find easy what he want to see on next. For example, to add two more options under “My List” with names “My List – Movies”, “My List - TV Shows” and filter them by duration info (only movies report the duration)

**Comment by CastagnaIT** [comment]
not within the addon because the “My List” data is paginated,
you can have something similar to this by using kodi library and enabling related features to "sync kodi library with my list" from the addon settings, in this case suggest you to read wiki

**Reply by tsioukas** [reply]
Thank you, you are right. I didn't think the pagination problem. I thought that you are retrieving from Netflix the full “My List” items so you can parse them into two new separated lists. I will try what you suggest, but basically, I use Netflix My List items as “bookmarks” on what I want to see in future.
